### DISCUSSION TITLE
Is there any way to donate to anime.js?

### DISCUSSION THREAD

**Discussion by cameralibre** [juliangarnier/anime#1003] [General]
I like to kick in a couple of bucks a month for the open source projects that I use regularly, and I love anime.js!

Is there (or is there any plan for) a patreon / open collective / tipeee / paypal link / github sponsors or something else? I'm sure there are many other community members who would be keen to help out in some way.

**Comment by LukeTOBrien** [comment]
@juliangarnier 
I have seen some Github repos with https://donorbox.org/ button on them, you could put it on your homepage.  
https://ko-fi.com/ is another one which is free and easy.

**Comment by juliangarnier** [comment]
Thanks!
I'm considering allowing donations to support the project, I might try GitHub Sponsors for that.

**Comment by cameralibre** [comment]
That would be great! Looking forward to signing up and spreading the word :)

**Comment by yellow1912** [comment]
Please allow sponsor, I think this lib is very well used and you deserve some tokens of appreciation.

**Comment by juliangarnier** [comment]
Finally!

https://github.com/sponsors/juliangarnier

And V4 is here too :)

**Comment by cameralibre** [comment]
signed up! nice.

**Comment by juliangarnier** [comment]
Moving this to Discussions!
